apps/wallet/src/ui/app/staking/validators/ValidatorsCard.tsx

```diff
@@ -44,10 +44,21 @@ export function ValidatorsCard() {
         return delegatedStake?.flatMap((delegation) => {
             return delegation.stakes.map((d) => ({
                 ...d,
+                // flag any inactive validator for the stakeSui object
+                // if the stakingPoolId is not found in the activeValidators list flag as inactive
+                inactiveValidator: !activeValidators?.find(
+                    ({ stakingPoolId }) =>
+                        stakingPoolId === delegation.stakingPool
+                ),
                 validatorAddress: delegation.validatorAddress,
             }));
         });
-    }, [delegatedStake]);
+    }, [activeValidators, delegatedStake]);
+
+    // Check if there are any inactive validators
+    const isInActiveValidator = delegations?.some(
```

Comment:
Maybe `hasInactiveValidatorDelegation` to clear this terminology up? "inactive" and "in active"  make variable naming hard